README.md

```diff
@@ -0,0 +1,87 @@
+# Music Library 🎶
+
+**First project with [Manchester Codes](https://www.manchestercodes.com/) BACKEND module**
+
+The purpose of this project was to apply all learned aspects together in one project by building an API (Application Programming Interface) app to manage a database, a music library.
+
+The project followed TDD principles by writing a failing test and write the code to pass the test. This time we used Mocha supertest, an integration test which checks that app is communicating with the database.
+
+---
+
+**Consepts**
```

Comment:
```suggestion
**Concepts**
```

Reply:
Lots of spelling mistakes in the Readme file. Teaches me to take time to proofread!